woocommerce/woocommerce-android
# Remove sale end button

Fixes #2110.

![image](https://user-images.githubusercontent.com/1522856/77548840-63bec400-6eaf-11ea-8021-873d54a18fab.png)

### To test:

1. Go to Products
2. Tap on one with a price
3. Tap on the Price section
4. Enable the Schedule sale section
5. **Notice the To date is empty** 
6. Add a date using a date picker
7. Tap on the Done button
8. **Notice the price sale date range is set in the Price section**
9. Go back to the Price screen
10. Tap on the Remove end date link
11. **Notice the To date is removed**
12. Tap on the Done button
13. **Notice the price sale only has the start date in the Price section**
14. Tap on the Update button
15. Go back to the product list
16. Tap on the same product and open the detail
17. **Notice the price sale dates have been preserved**

Update release notes:
